packages/icon/src/Icon.tsx

```diff
@@ -62,6 +63,7 @@ export const createIcon = (children: ReactNode, name: string) => {
 				role="img"
 				style={style}
 				className={className}
+				focusable={focusable}
```

Comment:
I found this bug while adding these tests. When `aria-hidden` is turned of, we should automatically turn this on. https://html-validate.org/rules/svg-focusable.html